examples/z_put.py

```diff
@@ -60,6 +63,14 @@
     conf.insert_json5(zenoh.config.LISTEN_KEY, json.dumps(args.listen))
 key = args.key
 value = args.value
+attachment = args.attach
+if attachment is not None:
+    attachment = {
+        k: v
+        for pair in attachment.split("&")
```

Comment:
The separator should be `;`.

```suggestion
        for pair in attachment.split(";")
```

Reply:
https://github.com/eclipse-zenoh/zenoh/blob/cbeffcf72649855d43b23893b4c8d499623cabc4/examples/examples/z_pub.rs#L42

Reply:
Right, the `;` is being *introduced* in protocol changes. Dropping this suggestion then.